src/fraud_credit_cards/data_processor.py

```
@@ -0,0 +1,36 @@
+        target = self.config['target']
+        self.X = self.df.drop(target, axis=1)
+        self.y = self.df[target]
+
+        # Define numeric features (remove categorical columns)
+        numeric_features = self.X.select_dtypes(include=['int64', 'float64']).columns.tolist()
+
+        # Define preprocessing steps
+        numeric_transformer = Pipeline(steps=[
+            ('scaler', StandardScaler())])
+
+        preprocessor = ColumnTransformer(
```

Ruff (0.8.2)

Local variable `preprocessor` is assigned to but never used

Remove assignment to unused variable `preprocessor`

(F841)